assets/src/amp-validation-error-detail-toggle.js

```diff
@@ -54,7 +61,20 @@ function addToggleListener() {
 	} );
 }
 
+function addViewErrorsByTypeLinkButton() {
+	if ( 'undefined' === typeof errorIndexAnchor || 'undefined' === typeof errorIndexLink ) {
+		return;
+	}
+	const heading = document.querySelector( '.wp-heading-inline' );
+	const link = document.createElement( 'a' );
+	link.innerHTML = errorIndexAnchor;
+	link.setAttribute( 'href', errorIndexLink );
+	link.setAttribute( 'class', 'page-title-action' );
+	heading.after( link );
+}
```

Comment:
Either this function or `AMP_Validation_Error_Taxonomy::render_link_to_errors_by_url()` should be refactored to be taking the same approach.